pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/rest/WritableResource.java

```diff
@@ -159,20 +159,16 @@ private static Response writeResponse(Bridge bridge,
                 ++totalWritten;
             }
         } catch (ClientAbortException e) {
-            LOG.debug("Remote connection closed by HAWQ", e);
+            LOG.error("Remote connection closed by HAWQ", e);
```

Comment:
this was not considered an error, for some reason, we are not throwing exception in this case, so maybe warning level ?